packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -174,7 +174,10 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         // SAFETY:    While the `attack` path does not need an extra check for the post
         //            state's depth in relation to the parent, we don't need another
         //            branch because (n - n) % 2 == 0.
-        bool validStep = VM.step(_stateData, _proof) == Claim.unwrap(postState.claim);
+        // TODO(clabby): Once output bisection is implemented, the local context will no longer
```

Comment:
Can we reference an issue number for the TODO rather than just you - then we know it should have been done if that issue is closed.